test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3937,7 +3937,7 @@ def log_lines(self, logtype, id=None, n=50, tail=True, starttime=None,
                     sudo = True
                 elif (isinstance(self, Scheduler) and
                         'sched_log' in self.attributes):
-                    # if setup is multi-sched the get log dir from
+                    # if setup is multi-sched the get logdir from
```

Comment:
typo 'the get' should be  'then get"

Reply:
Yeah. Sorry I didn't looked at it early.